docs/tutorials/3d-concepts-for-2d-web-developers.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 3D Concepts for Web Developers
```

Comment:
I think the concepts you are explaining can be explained in the context of XR.
Coordinate systems are very strongly related to XR Spaces. Currently you explain it like there's only one coordinate system, but for XR development its important to understand that there are many coordinate systems. Cameras are also a concept that is strongly related to XR. Having a document that first explains all the "just 3D" concepts, which whom we have to later correct them because they are different in XR might be weird for the reader. Therefore how about calling this "Introduction to AR and VR development" + "A quick introduction on basic 3D and AR/VR concepts for developers" and always relating those concepts to XR